### DISCUSSION TITLE
Is it possible to use an alias name for an existing theme color?

### DISCUSSION THREAD

**Discussion by seyaobey** [chakra-ui/chakra-ui#3354] [Q&A] [answered]
While extending the default theme, is it possible to alias an existing theme color with another name? For example is it possible to add a custom color (say `primary`) which is in fact the `teal.500` color of the default theme. Here is an illustration:

```
export const myTheme = extendTheme({
  ...
  colors: {
      primary: 'teal.500' // <-- alias teal.500 with primary
  }
})
```

I was expecting to see, in my output html, the real color value `#319795` (which is the teal.500 color). But I get this invalid style `color: teal.500`. Is such scenario possible? Thanks

**Comment by dodas** [comment, marked as answer]
Hello @seyaobey.
You can do that by importing the default theme and referencing a value:
```ts
import { extendTheme, theme } from "@chakra-ui/react"

export const myTheme = extendTheme({
  colors: {
    primary: theme.colors.teal[500],
  },
})
```
